analysers/trials/ram.py

```diff
@@ -15,22 +15,23 @@
 from pyspark_cassandra import RowFormat
 from pyspark import SparkConf
 
-def absoluteRamEfficency(sc, cassandraKeyspace, trialID, experimentID, containerID, hostID):
-    try:
-        containerProperties = sc.cassandraTable(cassandraKeyspace, "container_properties") \
-                .select("mem_total") \
-                .where("trial_id=? AND experiment_id=? AND container_id=? AND host_id=?", trialID, experimentID, containerID, hostID) \
-                .first()
-        maxMemory = containerProperties["mem_total"]
-        if maxMemory == 0:
-            hostProperties = sc.cassandraTable(cassandraKeyspace, "host_properties") \
-                .select("mem_total") \
-                .where("host_id=?", hostID) \
-                .first()
-            maxMemory = hostProperties["mem_total"]
-        absoluteEfficency = dataIntegral/float(maxMemory*metrics["num_data_points"])
-    except:
-        absoluteEfficency = None
+def absoluteRamEfficency(sc, cassandraKeyspace, trialID, experimentID, containerID, hostID, dataIntegral):
```

Comment:
@VincenzoFerme Docker writes the max memory as 0 or -1 when unlimited memory is available to the container, thus we need to go to the host properties to see the memory. Commenting to specify.